packages/memory/README.md

```
@@ -0,0 +1,406 @@
+| `project` | Ongoing work context, decisions, goals | "Auth migration driven by compliance, not tech debt" |
+| `reference` | Pointers to external resources | "Pipeline bugs tracked in Linear project INGEST" |
+
+## Memory Scopes
+
+| Scope | Path | Use Case |
+|-------|------|----------|
+| `global` | `memory/global/` | Cross-project preferences and knowledge |
+| `project` | `memory/project/{slug}/` | Project-specific context |
+| `agent` | `memory/agent/{id}/` | Per-agent isolation |
+
+## Storage Backends
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)